pkg/cmd/cli/describe/describer.go

```diff
@@ -556,12 +557,32 @@ func (d *RouteDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
+	endpoints, err := d.kubeClient.Endpoints(namespace).Get(route.Spec.To.Name)
+	if err != nil {
```

Comment:
Tolerate errors during retrieval and display the information available along with a message describing the error.  Something is a lot better than nothing and things like authorization rights can affect the outcome here.